boards/shields/rpi_pico_lcd/doc/index.rst

```diff
@@ -2730,497 +366,59 @@ ResTouch LCD 3.5| board. They will be built with activated USB-CDC/ACM console.
 
```

Comment:
**praise (blocking):** Change paragraph to: _This samples and test applications aren't applicable on all boards. They will be built with activated USB-CDC/ACM console._

Reply:
done